analysis/measures_definition_pf_descriptive_stats.py

```diff
@@ -1,42 +1,61 @@
 from ehrql import INTERVAL, create_measures, months
+from ehrql.tables.raw.tpp import medications
 from ehrql.tables.tpp import (
     practice_registrations,
     patients,
+    clinical_events
 )
 
-from pf_dataset import pharmacy_first_event_codes
-from measures_definition_pf_medications import pharmacy_first_med_codes
-from measures_definition_pf_breakdown import pharmacy_first_conditions_codes, selected_events
+from pf_codelist_functions import get_pf_condition_codes
+from pf_variables_library import get_consultation_ids, get_consultationid_events
+from codelists import pharmacy_first_med_codelist, pharmacy_first_consultation_codelist, pharmacy_first_conditions_codelist
 
 measures = create_measures()
 measures.configure_dummy_data(population_size=1000)
+#measures.configure_disclosure_control(enabled=False)
 
 start_date = "2024-02-01"
 monthly_intervals = 9
 
 registration = practice_registrations.for_patient_on(INTERVAL.end_date)
 
-# Loop through all codes in each sublist of the dictionary to flatten the list ready for is_in commands to be used and have a list of pf_condition codes
-pf_condition_codelist = [code for sublist in pharmacy_first_conditions_codes.values() for code in sublist]
+# Function to call pf_condition_codelist
+pf_conditions_codes = get_pf_condition_codes(pharmacy_first_conditions_codelist)
```

Comment:
Can you remind me why we are doing this? Could we not just use the regular codelist?

Reply:
For some reason it wasn't working before, but you're right, it doesn't need to be looped through. Thanks!